### DISCUSSION TITLE
How remove underline when fa icons is onhover ?

### DISCUSSION THREAD

**Discussion by Alama32** [FortAwesome/Font-Awesome#19702] [General]
How remove underline when fa icons is onhover ?

When i go to with mouse on hover icon, an underline appear, how remove that?  text-decoration dont work !

**Comment by tagliala** [comment]
Hi, I cannot replicate, probably there is a third-party rule with `important`

https://jsfiddle.net/tagliala/k7heosdc/1/

Please open an issue with a reproducible test case if you think there is an issue with Font Awesome

[bug report template for web applications](https://github.com/FortAwesome/Font-Awesome/issues/new?assignees=&labels=bug%2Cneeds-triage&template=100_web_bug_report.yml&title=Bug%3A+)

**Reply by Alama32** [reply]
Hi, thanks for your speedy reply, but not work , i have tryed.

**Reply by Alama32** [reply]
I  dont undestand this problem. 
It's inherent in the icons font awesome, because on gray areas, it doesn't. It's the hover mode on the fa-icons that does that.
I tried with a RAW HTML element and with the WP Coder plugin.
Here is the page in footer,  you can go and see with the inspector.
[https://ideablog.be/](https://ideablog.be/)

**Reply by tagliala** [reply]
Thanks for the reproducible test case.

Here there are the offending rules:

![image](https://user-images.githubusercontent.com/556268/235309877-208ccd69-c3df-441e-9ec7-606fcde051bb.png)

**Reply by Alama32** [reply]
Thanks for your help, really great, I never would have imagined that it could have something to do with the footer.. I solved the problem with this simple

`#footer a { 
    border-bottom: none;
}`

Really, many thanks

**Comment by Alama32** [comment]
Apparently I only have the issue when using the code in a Raw HTML element for Wordpress
Live on an empty page, it works well, under the Playgrounds, it works well too.
